src/lib/mail.ts

```diff
@@ -0,0 +1,45 @@
+import queryString from 'query-string';
+import {
+  isMobile,
+  osVersion,
+  osName,
+  fullBrowserVersion,
+  browserName,
+  deviceType,
+  mobileVendor,
+  mobileModel,
+} from 'react-device-detect';
+
+interface MailToParams {
+  bcc?: string;
+  cc?: string;
+  subject?: string;
+  body?: string;
+}
+
+const teamEmail = 'dreamscholars-contact@googlegroups.com';
+export function genMailToLink(params: MailToParams): string {
+  return `mailto:${teamEmail}?${queryString.stringify(params)}`;
+}
+
+const mobileInfo = `
+Device Type: ${deviceType}
+Vendor: ${mobileVendor}
+Model: ${mobileModel}`;
+
+export const bugReport = `
+---------- DO NOT EDIT THIS SECTION ----------
```

Comment:
Btw, this section might be useful for the feature request section too. In case the feature is something specific to a device, browser or screen size.